travis/linkchecker.config

```diff
@@ -0,0 +1,13 @@
+[filtering]
+
+status=0
+warnings=0
+checkextern=1
+ignore=
+    ^mailto:
+    chtc.wisc.edu
+    jira.opensciencegrid.org
+    twiki.org
+    batlab.org
+    confluence.grid.iu.edu
```

Comment:
It's not clear why some of these are ignored.  Is it possible to add a few comments or examples?

In particular, seems like we would want to notice-'n'-nuke links to confluence!